src/tests/test_utils/common_test_utils/src/graph_comparator.cpp

```diff
@@ -720,7 +729,10 @@ Comparator::Result Comparator::compare(ov::Node* node1, ov::Node* node2, std::os
     const bool subgraph_nodes = subgraph1 && subgraph2;
 
     if (subgraph_nodes) {
-        const auto result = subgraph::compare_io(subgraph1, subgraph2, should_compare(CmpValues::SUBGRAPH_DESCRIPTORS));
+        const auto result = subgraph::detail::CompareSubGraphs{get_comparison_flags()}.compare(
```

Comment:
should we also leave "subgraph::compare_io(..." call here?

Reply:
No. It's [removed](https://github.com/openvinotoolkit/openvino/pull/18254/files#diff-83864fde941ba60c5f392c39234a4cf4d9611af3fcaf8630509ab5f496273edbL582-L586) by this PR.